### DISCUSSION TITLE
jvm/zijiema-zhiling

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#261] [Announcements]
# jvm/zijiema-zhiling

二哥的Java进阶之路，小白的零基础Java教程，从入门到进阶，JVM字节码指令详解

https://javabetter.cn/jvm/zijiema-zhiling.html

**Comment by Raceylee** [comment]
testException 的例子中，第 4 行字节码应该是 goto 16 (+12) 跳转到 return 呢

**Comment by lumole** [comment]
路漫漫其修远兮，吾将上下而求索。😭
